### DISCUSSION TITLE
What does this do?

### DISCUSSION THREAD

**Discussion by cococoot** [trakt/showly#553] [Q&A]
![IMG_20211220_233412](https://user-images.githubusercontent.com/61015333/146872310-ab53a1f6-e0a4-4e10-abe2-829098f478ea.jpg)

**Comment by michaldrabik** [comment]
It's a button to toggle calendar modes. You can also pull down to trigger the same action.

**Reply by conradseba** [reply]
And what does this mean?
Because I cannot find the equivalent in Trakt.
Thanks!
![msg8205417-31902](https://user-images.githubusercontent.com/10448765/149681949-67351991-50c9-4386-8bf3-440c757681fe.jpg)

**Reply by michaldrabik** [reply]
When you remove episode/season/show from My Shows in the Showly app you can also remove it from your Trakt.tv account history ( https://trakt.tv/users/USERNAME/progress ) 

This can be turned of via app's settings.

**Reply by conradseba** [reply]
Yes, thanks. But when I remove a show clicking in "added to my shows", it does not remove my watched episodes, so what does it does in Trakt? Because there is no Trakt option to "add to my shows", AFAIK. That's my question, apologies if I wasn't/ain't clear enough. Thanks again.
